infrastructure/lambda-generate-document-manifest.tf

```diff
@@ -78,11 +78,12 @@ module "generate-document-manifest-lambda" {
     module.zip_store_reference_dynamodb_table,
     module.ndr-zip-request-store,
     module.ndr-document-store,
-    module.ndr-lloyd-george-store
+    module.ndr-lloyd-george-store,
+    aws_iam_policy.dynamodb_stream_manifest_policy
   ]
 }
 
-resource "aws_iam_policy" "dynamodb_stream_policy" {
+resource "aws_iam_policy" "dynamodb_stream_manifest_policy" {
```

Comment:
If we're renaming this already, can it be called `dynamodb_stream_manifest` instead?